tests/RPCClient.test.ts

```diff
@@ -292,26 +301,17 @@ describe(`${RPCClient.name}`, () => {
       >(methodName);
       await callerInterface.writable.close();
       const callProm = (async () => {
-        for await (const _ of callerInterface.readable) {
-          // Only consume
-        }
+        // Only consume values
+        for await (const _ of callerInterface.readable);
       })();
       await expect(callProm).rejects.toThrow(rpcErrors.ErrorRPCRemote);
       await outputResult;
     },
   );
-  test.prop({
-    messages: fc.array(rpcTestUtils.JSONRPCResponseSuccessArb()),
-    errorMessage: rpcTestUtils.JSONRPCResponseFailedArb(
-      rpcTestUtils.errorArb(),
-    ),
-  })(
-    'generic duplex caller can throw received error message with sensitive',
-    async ({ messages, errorMessage }) => {
-      const inputStream = rpcTestUtils.messagesToReadableStream([
-        ...messages,
-        errorMessage,
-      ]);
```

Comment:
The 'throw error message with sensitive' was basically a duplicated test, so I removed it. It's hard to see here due to how github is rendering the diff, so I would recommend seeing the test in the actual file via staging.